docs/running.md

```diff
@@ -9,14 +9,224 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
 
```

Comment:
Can we move this to a "Logging" section (maybe even in another file in our documentation).

> If the `$XDG_STATE_HOME` environment variable is set, the server will log to
> a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet/warnet.log`.

The reason why I think that it's important to move this information is because new users will want to hyper focus on getting their system running which means that this could distract them. Feels like premature configuration.